modules/swagger-codegen/src/main/java/io/swagger/codegen/CodegenConstants.java

```diff
@@ -41,7 +41,7 @@
     public static final String SORT_PARAMS_BY_REQUIRED_FLAG_DESC = "Sort method arguments to place required parameters before optional parameters.";
 
     public static final String ENSURE_UNIQUE_PARAMS = "ensureUniqueParams";
-    public static final String ENSURE_UNIQUE_PARAMS_DESC = "Whether to ensure parameter names are unique in an operation (rename parameters that are not). Default: true";
+    public static final String ENSURE_UNIQUE_PARAMS_DESC = "Whether to ensure parameter names are unique in an operation (rename parameters that are not).";
```

Comment:
shall we keep "Default: true"" so that users know the default value (true) for `ensureUniqueParams` ?